docs/freeipa.md

```diff
@@ -0,0 +1,141 @@
+# FreeIPA
+
+FreeIPA is a product that provides (not exhaustive):
+
+- CA
+- KDC
+- LDAP
+- Host management (including SSSD)
+
+The playbook is able to provision a FreeIPA server, or use an existing FreeIPA server, and automatically setup the cluster to make use of the CA, KDC and LDAP.
+
+Before continuing, a few questions you may want to ask yourself are:
+
+1. Are you using an existing FreeIPA server(s) or do you want the playbook to provision one?
+2. Do you want to use the FreeIPA CA to sign the certificates or do you want to sign them externally?
+3. Are you using AutoTLS or do you want the playbook to configure the host keys and certificates?
+
+## Common steps
+
+Regardless of how you choose to use FreeIPA in your deployment, you'll have to set the following variables via extra vars:
```

Comment:
better to say in the definition than extra vars

Reply:
Fixed